protos/protos/public/modeldb/metadata/MetadataService.proto

```diff
@@ -12,14 +12,22 @@ message IDTypeEnum {
         UNKNOWN = 0; // Type 0 should always cause an error
         VERSIONING_REPOSITORY = 1;
         VERSIONING_COMMIT = 2;
+        VERSIONING_REPO_COMMIT_BLOB = 3;
     }
 }
 
+message VersioningCompositeIdentifier {
```

Comment:
Can we just use a string? The idea of this was exactly to avoid having to add one entry (that I believe will map to database changes and a ton of other overhead) every time we want to add something.

Reply:
db entry is still a string ,  this is just the representation over wire.